docs/source/user_guide/installation/gnu-linux-mac.rst

```diff
@@ -64,27 +65,25 @@ the environment and go back to your original system, just type:
 
    deactivate
 
-PyBaMM can be installed via pip. On macOS, it is necessary to install the `SUNDIALS <https://computing.llnl.gov/projects/sundials/>`__
+PyBaMM can be installed via ``pip`` or ``conda``. On macOS, it is necessary
+to install the `SUNDIALS <https://computing.llnl.gov/projects/sundials/>`__
```

Comment:
```suggestion
PyBaMM can be installed via ``pip`` or ``conda``.
```

This line has been in our installation guide (and hasn't been needed) since a long time, ever since we started publishing wheels with the SUNDIALS libs bundled with them.